service/controller/v21/templates/cloudformation/guest/vpc.go

```diff
@@ -23,4 +23,26 @@ const VPC = `{{define "vpc"}}
       Tags:
         - Key: Name
           Value: {{ $v.ClusterID }}
+  VPCS3Endpoint:
+    Type: 'AWS::EC2::VPCEndpoint'
+    Properties:
+      VpcId: !Ref VPC
+      RouteTableIds:
+        {{- range $v.RouteTableNames }}
+        - !Ref {{ .ResourceName }}
+        {{- end}}
+      ServiceName: 'com.amazonaws.{{ $v.Region }}.s3'
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: "{{ $v.ClusterID }}-vpc-s3-endpoint-policy-bucket"
```

Comment:
What is `Sid`? I don't see it in examples. Some arbitrary name?

Reply:
`Sid` is an unique ID  that you apply to the policy to be uniquely identified and displayed in the UI. it is simply useful for displaying. It can be left blank. It is not mandatory

Reply:
Its inline policy name, its not required, but it helps explain what is that policy for